packages/terra-paginator/tests/wdio/paginator-spec.js

```diff
@@ -125,4 +125,20 @@ Terra.describeViewports('Controlled Paginator', ['large'], () => {
       Terra.validates.element('progressive without total item count');
     });
   });
+
+  describe('Paginator Button should not retain focused state once disabled using keyboard navigation', () => {
+    it('should navigate to next button and enter', () => {
+      browser.url('/raw/tests/terra-paginator/paginator/controlled-progressive-paginator');
+      browser.keys(['Tab', 'Tab', 'Tab', 'Tab', 'Enter']);
+    });
+
+    it('should navigate to previous button and enter', () => {
+      browser.keys(['Shift', 'Tab', 'Shift', 'Enter']);
+    });
+
+    it('should navigate back to next button and enter', () => {
+      browser.keys(['Tab', 'Enter']);
+      Terra.validates.element('previous button is not focused');
+    });
```

Comment:
this might be nitpicky, but these first two it blocks are really setup for the actual test assertion. My vote is to convert these into a single it block. We could add a comment before each `browser.keys` to specify what setup we are doing. I'm curious what you think?

Reply:
oops, I should have specified that I was thinking we could combine them into the last it block that does the `Terra.validates` call. Not a big deal tho

Reply:
I will do that.

Reply:
Update here https://github.com/cerner/terra-core/pull/3438/commits/8058e9b945f88eba76a1ceed0b8f2b71dadd6a08